MediaProcessor/src/ConfigManager.cpp

```diff
@@ -29,4 +29,11 @@ std::string ConfigManager::getFFmpegPath() const {
     return m_config["ffmpeg_path"].get<std::string>();
 }
 
+int ConfigManager::getNumChunks() const {
```

Comment:
Added a New Funtion to get the num_chunks from config File

Reply:
As explained below, please rename to `getNumThreadsValue`, as to avoid chunks in the name, which is internal to AudioProcessor, as well as to adhere to convention of value for key.